phpBB/phpbb/search/backend/base.php

```diff
@@ -289,4 +283,124 @@ function destroy_cache($words, $authors = false)
 			WHERE search_time < ' . (time() - (int) $config['search_store_results']);
 		$db->sql_query($sql);
 	}
+
+	/**
+	 * {@inheritdoc}
```

Comment:
Where is this supposed to inherit from? The abstract class does not implement an interface.